auto_submit/lib/request_handling/pubsub.dart

```diff
@@ -35,4 +36,30 @@ class PubSub {
     final PublishResponse response = await pubsubApi.projects.topics.publish(request, _topic);
     log.info('pubsub response messageId=${response.messageIds}');
   }
+
+  /// Pulls messages from the server.
+  Future<PullResponse> pull(int maxMessages, String subscription) async {
```

Comment:
This should align with the publish api, and do `subscription, maxMessages`

Reply:
Gotcha, changed the order of parameters.